From 2428abb8dd4bc3c176a91a6d53d13b8b0cc67e24 Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Wed, 15 Jun 2022 12:47:23 +0300
Subject: [PATCH] arm64: meson: add JetHub D1p device

Add support for JetHub D1p variation of D1 series home automation controller.
---
 .../dts/amlogic/meson-axg-jethome-jethub-j100.dts   | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts b/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
index 8b0d586aa84e..f46b0ec9ec13 100644
--- a/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
@@ -258,18 +258,14 @@ &sd_emmc_b {
 	bus-width = <4>;
 	cap-sd-highspeed;
 	max-frequency = <50000000>;
-	non-removable;
 	disable-wp;
+	keep-power-in-suspend;
+	broken-cd;/* cd-gpios = <&gpio GPIOAO_3 GPIO_ACTIVE_LOW>;*/
 
 	mmc-pwrseq = <&sdio_pwrseq>;
 
 	vmmc-supply = <&vddao_3v3>;
 	vqmmc-supply = <&vddio_boot>;
-
-	brcmf: wifi@1 {
-		reg = <1>;
-		compatible = "brcm,bcm4329-fmac";
-	};
 };
 
 /* emmc storage */
@@ -368,3 +364,8 @@ &cpu2 {
 &cpu3 {
 	#cooling-cells = <2>;
 };
+
+&saradc {
+	status = "okay";
+	vref-supply = <&vddio_ao18>;
+};
-- 
2.30.2

